components/inputs/input-textarea.js

```diff
@@ -147,6 +147,26 @@ class InputTextArea extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))
 		this._textareaId = getUniqueId();
 	}
 
+	get textarea() {
```

Comment:
I must admit, I'm not a super big fan of moving these accessors between the `constructor` and the Lit methods, but I agree that the configuration before Krishan's change was already doing this for read/write accessors, and that changing the order either way is bound to be disruptive. 🤷‍♂️